mycli/main.py

```diff
@@ -582,6 +583,34 @@ def handle_clip_command(self, text):
             return True
         return False
 
+    def handle_prettify_binding(self, text):
+        try:
+            statements = sqlglot.parse(text)
+        except:
+            statements = []
+        if len(statements) == 1:
+            pretty_text = statements[0].sql(pretty=True, pad=4)
+        else:
+            pretty_text = ''
+            self.bell()
```

Comment:
Dinging a bell for this seems wee bit intrusive. I can be convinced otherwise.

Reply:
Perhaps we can add a config option to ding the bell or not and read that in the self.bell() function so it will be centralized?

Reply:
I agree it is too intrusive.  It was just the only thing I could figure out in a draft.  A message would be better.

Reply:
resolved